plugins/kafka/CMakeLists.txt

```diff
@@ -27,58 +27,74 @@ project(kafka)
 if(KAFKA)
     ADD_PLUGIN(kafka)
     if(MAKE_KAFKA)
-        # librdkafka packages do not include everything we need to properly
-        # build against it; until/if they are ever fixed we will need to build
-        # our own version of librdkafka from source and include it ourselves
-
-        if(NOT EXISTS "${PROJECT_SOURCE_DIR}/librdkafka/configure")
-            message(FATAL_ERROR
+        if (VCPKG_APPLOCAL_DEPS)
+            find_package(RdKafka REQUIRED)
+
+            set ( LOCAL_LIBRDKAFKA_LIBS 
+                RdKafka::rdkafka 
+                RdKafka::rdkafka++
+            )
+        else ()
```

Comment:
This "if/else" block could be reworked so it is self contained in commonSetup.cmake (see blas support).

Reply:
...actually it can't as its a "plugin" but it could be cleaned up a little.

Reply:
...also not true.